NotificationBanner/Classes/BaseNotificationBanner.swift

```diff
@@ -580,14 +584,13 @@ open class BaseNotificationBanner: UIView {
     }
 
     /**
-         Determines wether or not we should adjust the banner for notch featured iPhone
+        Basic height that is used unless a custom height is provided
      */
 
-    internal func shouldAdjustForNotchFeaturedIphone() -> Bool {
-        return NotificationBannerUtilities.isNotchFeaturedIPhone()
-            && UIApplication.shared.statusBarOrientation.isPortrait
-            && (self.parentViewController?.navigationController?.isNavigationBarHidden ?? true)
+    internal func basicHeight() -> CGFloat {
+        return 64.0 + basicYOffset
```

Comment:
Another magic number. Could we use a constant or calculate this value?

Reply:
Moved to a var as well